core/res/res/values/ex_config.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     Copyright (C) 2021 AospExtended ROM
```

Comment:
ex configs should be in 404_config anyway we will add it to source thanks for contribution :)